CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Contributing to Stencil
+
+Thanks for showing interest in contributing!
+
+The following is a set of guidelines for contributing to Stencil. These are just guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+By contributing to Stencil, you agree that your contributions will be licensed under its MIT license.
+
+#### Table of Contents
+
+[Stencil Documentation](https://stencil.bigcommerce.com/docs)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+  * [Git Commit Messages](#git-commit-messages)
+  * [JavaScript Styleguide](#javascript-styleguide)
+  * [CSS/Sass Styleguide](#css-styleguide)
+
+### Your First Code Contribution
+
+Unsure where to begin contributing to Stencil? Check our [forums](https://forum.bigcommerce.com/s/group/0F91300000029tpCAA) or our [stackoverflow](https://stackoverflow.com/questions/tagged/bigcommerce) tag. 
+
+### Pull Requests
+
+* Fill in [the required template](PULL_REQUEST_TEMPLATE.md)
+* Include screenshots and animated GIFs in your pull request whenever possible.
+* End files with a newline.
+
+## Styleguides
+
+### Git Commit Messages
+
+* Use the present tense ("Add feature" not "Added feature")
+* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
```

Comment:
One helpful tip I use when writing commit messages is making sure they complete the sentence: "If merged, this commit will.." It sounds like commits that meet that requirement will also meet these, so adding it could be helpful to those who aren't familiar with the terms present tense and imperative mood.